python/private/stage2_bootstrap_template.py

```diff
@@ -66,7 +66,7 @@ def get_windows_path_with_unc_prefix(path):
             break
         except (ValueError, KeyError):
             pass
-    if win32_version and win32_version >= '10.0.14393':
+    if win32_version and win32_version >= "10.0.14393":
```

Comment:
This line remembered me someone posting on #2652, I wonder if this PR would address that as well, where Windows is suffering hitting `MAX_PATH`.